pkg/testing/integration/program.go

```diff
@@ -1265,11 +1265,6 @@ func (pt *ProgramTester) TestLifeCycleInitialize() error {
 	dir := pt.projdir
 	stackName := pt.opts.GetStackName()
 
-	// If RelativeWorkDir is specified, apply that relative to the temp folder for use as working directory during tests.
```

Comment:
Isn't this needed for the various stack config commands ran below?

Reply:
`pt.projdir` is set to incorporate `RelativeWorkDir` in [`copyTestToTemporaryDirectory()`](https://github.com/pulumi/pulumi/pull/12758/files/5c24b324292b55cb4d4f45728c02974384cc9b08#diff-58bdc2879a0109885b20b1ac1c52e30423b544b4a0b3e4326c1b79a063c7ce14R1771), by the time `dir` is set, `RelativeWorkDir` is already respected IIRC.